app/src/main/java/com/morse/App.java

```diff
@@ -45,4 +51,43 @@ public void checkCredentials(Channel channel) {
         // TODO implement here
     }
 
+    private void createDB() {
+        String url = "jdbc:sqlite:app/src/main/database.db";
+        try (Connection conn = DriverManager.getConnection(url)) {
+            if (conn != null) {
+                System.out.println("Database created!");
+            }
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private void createTable() {
+        String url = "jdbc:sqlite:app/src/main/database.db";
+        String sqlCommand = "CREATE TABLE IF NOT EXISTS accounts (\n"
+                + "	channel text PRIMARY KEY,\n"
+                + "	username text NOT NULL,\n"
+                + "	password text NOT NULL\n"
+                + ");";
+        try (Connection conn = DriverManager.getConnection(url)) {
+            Statement statement = conn.createStatement();
+            statement.execute(sqlCommand);
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private void insertIntoTable(String channel, String userName, String hashedPassword){
+        String url = "jdbc:sqlite:app/src/main/database.db";
+        try (Connection conn = DriverManager.getConnection(url)) {
+            Statement statement = conn.createStatement();
+            statement.execute("INSERT INTO accounts(channel, username, password) VALUES(" +
+                    "\"" + channel + "\"" + ", " +
+                    "\"" + userName + "\"" + ", " +
+                    "\"" + hashedPassword + "\"" +");");
```

Comment:
I don't know if this matters that much, but this is prone to sql injection. Use parameterized queries.